inthe_am/taskmanager/serializers/task.py

```diff
@@ -15,6 +15,7 @@ class TaskSerializer(serializers.Serializer):
     modified = serializers.DateTimeField(read_only=True)
     start = serializers.DateTimeField(required=False)
     wait = serializers.DateTimeField(required=False)
+    until = serializers.DatetimeField(required=False)
```

Comment:
Whoops, missed this when reviewing this; Python is case-sensitive, so this has to be `DateTimeField`. 
 I've manually fixed this in 6305cc8565b253c2d0f0c12230a0a70286d00e65, though, so don't worry about updating the PR.